src/main/scala/jp/ne/opt/redshiftfake/parse/CopyCommandParser.scala

```diff
@@ -61,6 +61,8 @@ class CopyCommandParser extends BaseParser {
     s"$any*(?i)BZIP2".r
   }
 
+  private[this] val ignoreHeaderParser = s"$any*(?i)IGNOREHEADER$space+AS".r ~> "'" ~> """[^']*""".r <~ "'" <~ s"$any*".r
```

Comment:
- According to Redshift document, `AS` is optional so let's make it omittable
- Does the number-of-rows need to be surrounded by single quote? I don't have an environment to test it but according to the document, seems it's not necessary

Reply:
Thanks for coming back and reviewing the PR, @ocadaruma . Great observations 👍 
* I have now updated the `UNLOAD` command parser to interpret the `AS` keyword as optional
* Indeed, you're right. I have changed the `IGNOREHEADER` parser now to accept an unquoted integer, rather than a quoted string for the _number of rows_ value

Let me know if you have any additional questions or comments. All tests are passing now. If you're happy with the changes, could you please merge the PR and release a new version? Thanks 🙇‍♂️